### DISCUSSION TITLE
I can't save the Ollama Base URL :(

### DISCUSSION THREAD

**Discussion by gerroon** [open-webui/open-webui#1685] [Q&A] [answered]
Hi

I add my local Ollama in the base url entry in the settings, press save. I then go back to the same settings it is empty. I do not get it. Is there another way to save it?

Wsl Docker

```
Open WebUI Version
v0.1.120

```

![image](https://github.com/open-webui/open-webui/assets/8519469/ad3240b6-780e-40f3-aefc-9a3a95487a24)

**Comment by justinh-rahb** [comment, marked as answer]
Environment variable:
`OLLAMA_BASE_URL=<your-ollama-url>`

For example if your Ollama host is on http://192.168.1.100 you'd use:
```bash
docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=https://192.168.1.100:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```

**Reply by gerroon** [reply]
Hey thanks for the answer but it does not work. It keeps resetting to this after restarting the container `http://host.docker.internal:11434`

cat .env
```
# Ollama URL for the backend to connect
# The path '/ollama/api' will be redirected to the specified backend URL
OLLAMA_API_BASE_URL='http://192.168.5.1:11434/api'
OLLAMA_BASE_URL='http://192.168.5.1:11434'
```

Docker run
```
docker run -d -p 11080:8080 -e OLLAMA_API_BASE_URL=http://192.168.5.1:11434/api --env https_proxy=http://192.100.1.1:8118 --env http_proxy=http://192.100.1.1:8118 -v /media/drive/ollama_webui/data:/app/backend/data --restart always --name ollama-webui ghcr.io/open-webui/open-webui:main
```

**Reply by tjbck** [reply]
Not sure what the other env vars are, try:

```sh
docker run -d -p 11080:8080 -e OLLAMA_BASE_URL=http://192.168.5.1:11434 -v /media/drive/ollama_webui/data:/app/backend/data --restart always --name open-webui ghcr.io/open-webui/open-webui:main
```

**Reply by spammenotinoz** [reply]
Same issue with OPENAI API Key, but as I am a noob to this project could be user error.
